hr_holidays_working_time/models/hr_holidays.py

```diff
@@ -55,7 +55,8 @@ def _auto_init(self, cr, context=None):
     number_of_hours = fields.Float(
         'Number of Hours', compute='_compute_number_of_hours', store=True)
     number_of_days_temp = fields.Float(
-        compute='_compute_number_of_hours_from_hours', store=True)
+        compute='_compute_number_of_hours_from_hours', store=True,
+        required=False)
```

Comment:
So, it's an Odoo bug ?

BTW, We have to be sure that there is a value in this field. So, it's necessary to add a constraint.